environs/config/config.go

```diff
@@ -952,6 +972,14 @@ func (c *Config) MaxStatusHistorySizeMB() uint {
 	return uint(val)
 }
 
+// UpdateStatusHookInterval is how often to run the charm
+// update-status hook.
+func (c *Config) UpdateStatusHookInterval() time.Duration {
+	// Value has already been validated.
+	val, _ := time.ParseDuration(c.mustString(UpdateStatusHookInterval))
```

Comment:
I can't repro a scenario where this is an issue, but none the less I've added a second lot of default handling in the getter method. We should remove it once 2.1.x becomes EOL. Previously it was controller config so that could explain any corner case there may have been.